crates/oci-distribution/src/client.rs

```diff
@@ -880,9 +887,13 @@ mod test {
 
     #[test]
     fn manifest_url_generation_respects_http_protocol() {
-        let c = Client::new(ClientConfig {
-            protocol: ClientProtocol::Http,
-        });
+        let c = Client::new(
+            ClientConfig {
+                protocol: ClientProtocol::Http,
+            },
+            None,
+        )
+        .expect("Could not crate Client");
```

Comment:
typo: create*

Reply:
Now resolved, that's what happens when you do copy and paste ;)